### DISCUSSION TITLE
How to customize layout of GmapInfoWindow by the position of marker in vue2-google-maps

### DISCUSSION THREAD

**Discussion by prodev1107** [vuetifyjs/vuetify#18757] [Q&A] [answered]
I have a trouble in the vue2-google-maps.
Like google maps, when the marker is in the left side of the screen, I want to set GmapInfoWindow to the right side of the marker.
And in reverse, when the marker is in the top side of the screen, I want to set GmapInfoWindow to the bottom side of the marker.
Who know the solution about this problem?

**Comment by nickmadev** [comment]
@prodev1106 
Could you share with me your current npm library link?

**Comment by prodev1107** [comment]
Thanks for your reply.
I am using vue2-google-maps.
https://www.npmjs.com/package/vue2-google-maps

**Comment by nickmadev** [comment, marked as answer]
I tried similar solution a month ago with your library.
I analyzed the code and found default arrow style option is always "down".
You can customize position, but arrow of your infoWindow is always "down side".

You have 2 options.
- First one is to customize this library, you can clone whole code of this library to your codebase, and implement dynamic arrows, this library is also using google map API.
You can do it.
- Second one is to auto reload of the map when the info-window is in the boundary.
You can expand the map automatically.

**Reply by prodev1107** [reply]
Got it, thanks for your answer.
